js/pento/pento_shape.js

```diff
@@ -183,11 +209,10 @@ $(document).ready(function () {
 			}
 			return i;
 		}
-
+		
+		//TODO: remove?
```

Comment:
do we know what it's supposed to do? If not, I vote for removing

Reply:
ah, it's actually called further down, but if it has a default value, that's strange...